aries-site/src/layouts/navigation/NavPage.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box, Text, Paragraph } from 'grommet';
+import { iconsMap } from '../../components/icons/iconsMap';
+
+import { navItemsData } from './navItemsData';
+
+const NavItem = ({ item, prefix }) => {
+  const itemLowerCase = item.toLowerCase();
+  return (
+    <Box
+      fill
+      direction="row"
+      margin={{ vertical: 'large' }}
+      onClick={() => (window.location.href = `/${prefix}/${itemLowerCase}`)}
+      gap="large"
+    >
+      {iconsMap[itemLowerCase]('xlarge')}
+      <Box>
+        <Text weight="bold" size="large">
+          {item}
+        </Text>
+        <Paragraph size="small">
+          {navItemsData[prefix][itemLowerCase]}
+        </Paragraph>
+      </Box>
+    </Box>
+  );
+};
+
+export const NavPage = ({ items, prefix, ...rest }) => {
+  return (
+    <Box {...rest}>
+      {items.map(item => (
+        <span key={item}>
```

Comment:
General question: Semantically, should these be considered `<li>` and the parent `<Box as="ul">`?

Reply:
That would make sense to me.